src/kademlia/kademlia.ts

```diff
@@ -27,70 +26,156 @@ export class KademliaRoutingTable extends (EventEmitter as { new (): BucketEvent
    * @param localId the ID of the local node
    * @param k the size of each bucket (k value)
```

Comment:
Did you intend to leave this `k` param definition here since it was removed from the constructor?  Just curious as I was doing some work on my portalnetwork module and noticed this k parameter was no longer passed to the table constructor and it looks like you are no longer limiting the size of each bucket.  Is that accurate?

Reply:
It was meant to be hardcoded to a constant, looks like I missed this

Reply:
Ah, so is discv5 generally moving to a practice of fixed sized buckets? Just curious as it will have downstream effects (probably minimal but still impacts) on Ultralight

Reply:
This shouldn't have any downstream effects (unless you need to configure the size of the buckets differently).
But afaik everything uses the same size bucket.

Reply:
Okay, there haven't been any final decisions about bucket sizes yet for the portal network but will keep this in mind once we get there.